fronts-client/src/actions/Cards.ts

```diff
@@ -228,19 +224,15 @@ const isFlexibleGeneralContainer = (state: State, collectionId: string) => {
  * it should adopt the minimum boost level
  * of the group it moves into, regardless of its previous boost level.
  * */
-const mayResetBoostLevel = (
-	state: State,
+export const mayResetBoostLevel = (
 	from: PosSpec | null,
 	to: PosSpec,
 	card: Card,
 	persistTo: 'collection' | 'clipboard',
 ) => {
-	if (to.type !== 'group' || !to.collectionId || persistTo !== 'collection')
-		return;
+	if (to.type !== 'group' || persistTo !== 'collection') return;
 	if (from?.id === to.id) return;
-	if (!isFlexibleGeneralContainer(state, to.collectionId)) return;
-	const toGroup = selectGroups(state)[to.id];
-	const groupName = toGroup?.name ?? 'standard';
```

Comment:
We can access the group name from `to` so we don't need to make a separate check to retrieve it.